conf/.env

```diff
@@ -37,138 +55,274 @@ LOG_CHANNEL=daily
-# This makes it easier to migrate your database. Not that some fields will never be decrypted.
-USE_ENCRYPTION=false
+# The map will default to this location:
+MAP_DEFAULT_LAT=51.983333
+MAP_DEFAULT_LONG=5.916667
+MAP_DEFAULT_ZOOM=6
 
 # Firefly III has two options for user authentication. "eloquent" is the default,
-# and "adldap" for LDAP servers.
+# and "ldap" for LDAP servers.
 # For full instructions on these settings please visit:
-# https://firefly-iii.readthedocs.io/en/latest/installation/authentication.html
-LOGIN_PROVIDER=ldap
+# https://docs.firefly-iii.org/advanced-installation/authentication
+# If you use Docker or similar, you can set this variable from a file by appending it with _FILE
+LOGIN_PROVIDER=eloquent
+
+# It's also possible to change the way users are authenticated. You could use Authelia for example.
+# Authentication via the REMOTE_USER header is supported. Change the value below to "remote_user_guard".
+#
+# This will also allow Windows SSO.
+#
+# If you do this please read the documentation for instructions and warnings:
+# https://docs.firefly-iii.org/advanced-installation/authentication
+#
+# This function is available in Firefly III v5.3.0 and higher.
+AUTHENTICATION_GUARD=web
+
+# If the guard is changed, Firefly III uses the 'REMOTE_USER' header as per RFC 3875.
+# You can also use another header, like AUTH_USER when using Windows SSO.
+# Some systems use X-Auth headers. In that case, use HTTP_X_AUTH_USERNAME or HTTP_X_AUTH_EMAIL
+# Depending on your system, REMOTE_USER may need to be changed to HTTP_REMOTE_USER
+#
+AUTHENTICATION_GUARD_HEADER=REMOTE_USER
+
+#
+# Firefly III uses email addresses as user identifiers. When you're using an external authentication guard
+# that doesn't do this, Firefly III is incapable of emailing you. Messages sent to "Bill Gates" always fail.
+#
+# However, if you set this value, Firefly III will store the value from this header as the user's backup
+# email address and use it to communicate. So user "Bill Gates" could still have
+# the email address "bill@microsoft.com".
+#
+# Example value: AUTHENTICATION_GUARD_EMAIL=HTTP_X_AUTH_EMAIL
+#
+AUTHENTICATION_GUARD_EMAIL=
+
+
+# It's impossible to log out users who's authentication is handled by an external system.
+# Enter a custom URL here that will force a logout (your authentication provider can tell you).
+# Setting this variable only works when AUTHENTICATION_GUARD != web
+#
+CUSTOM_LOGOUT_URI=
 
 # LDAP connection configuration
 # OpenLDAP, FreeIPA or ActiveDirectory
+# # If you use Docker or similar, you can set this variable from a file by appending it with _FILE
 ADLDAP_CONNECTION_SCHEME=OpenLDAP
 ADLDAP_AUTO_CONNECT=true
 
 # LDAP connection settings
-ADLDAP_CONTROLLERS=localhost
+# You can set the following variables from a file by appending them with _FILE:
+# ADLDAP_CONTROLLERS, ADLDAP_PORT, ADLDAP_BASEDN
+ADLDAP_CONTROLLERS=
 ADLDAP_PORT=389
 ADLDAP_TIMEOUT=5
-ADLDAP_BASEDN="dc=yunohost,dc=org"
```

Comment:
Have you tested ldap authentication because usually BASEDN is something required

Reply:
This does nothing yet. Its moving a step closer to implementing it.